### PR TITLE
Updated some format strings for improved Python 3 compatibility #511

### PR DESCRIPTION
**Pull request by Onager**
Updated some format strings for improved Python 3 compatibility #511

### CODE REVIEW

File: plaso/multi_processing/plaso_xmlrpc.py [hunk 1]
```diff
@@ -47,7 +47,7 @@ def CallFunction(self):
     except (
         expat.ExpatError, SocketServer.socket.error,
         xmlrpclib.Fault) as exception:
-      logger.warning('Error while making RPC call: {0:s}'.format(exception))
+      logger.warning('Error while making RPC call: {0!s}'.format(exception))
```
[the remark this answers is not in the document]
Comment (Onager) [reply plaso/multi_processing/plaso_xmlrpc.py]: Done

File: plaso/parsers/winreg.py [hunk 2]
```diff
@@ -137,7 +137,7 @@ def _ParseKeyWithPlugin(self, parser_mediator, registry_key, plugin):
       plugin.UpdateChainAndProcess(parser_mediator, registry_key)
     except (IOError, dfwinreg_errors.WinRegistryValueError) as exception:
       parser_mediator.ProduceExtractionError(
-          'in key: {0:s} {1:s}'.format(registry_key.path, exception))
+          'in key: {0:s} {1!s}'.format(registry_key.path, exception))
```
[the remark this answers is not in the document]
Comment (Onager) [reply plaso/parsers/winreg.py]: Done

File: plaso/formatters/interface.py [hunk 3]
```diff
@@ -82,7 +82,7 @@ def _FormatMessage(self, format_string, event_values):
 
       error_message = (
           'unable to format string: "{0:s}" event object is missing required '
-          'attributes: {1:s}').format(format_string, exception)
+          'attributes: {1!s}').format(format_string, exception)
```
[the remark this answers is not in the document]
Comment (Onager) [reply plaso/formatters/interface.py]: "With error" is already in the message, per line 86

**Comment by joachimmetz** [comment]
@Onager please fix "This branch has conflicts that must be resolved"

**Pull request merged**